modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/OrganizationStagedModelDataHandler.java

```diff
@@ -168,6 +171,8 @@ protected void doImportStagedModel(
 
 		Organization importedOrganization = null;
 
+		organization = _importCountry(portletDataContext, organization);
+
```

Comment:
Why is there this and not with all the other _import*? (line 196). Is there any logic? Otherwise please move below with the others

Reply:
This is here since the countryID is being added and validated when the Organization is being added or updated (in lines 179 - 186). It gets checked early so it can't be with the other imports.

Reply:
Ok, can we move it on top of the method?